### DISCUSSION TITLE
Billing Code Types other than CPT, HCPCS, DRG, ICD, NDC-- how are they reported?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#9] [Q&A] [answered]
Currently, the Schema definitions on GitHub have specific 'data' values, where the rule indicates that a 'own indicator' can be used when a billing code is one that is not defined.  
For example, the only billing code(s) referenced are CPT, HCPCS, DRG, ICD, NDC.   If there were combinations, such as Revenue Code or something else---how would these be reported?  
The rule says to see details in the IG.   I am not seeing any examples on GitHub to provide guidance when a item or service is not one of the defined data values.

**Comment by shaselton-usds** [comment]
Would you have an example of what a Revenue Code some of other code that might not be able to fit the `billing_code_type`, `billing_code_type_version`, and `billing_code` combinations? Maybe what would those codes look like?

**Reply by vfohl** [reply]
If we have a contract based solely on revenue codes, how does that get reported since revenue code is not listed as a billing code type?

**Reply by shaselton-usds** [reply]
Is it typical for a revenue code to have an acronym -- maybe something like RC? Like cognitive processing therapy is CPT.

Also, how are revenue code's versioned?

**Reply by tdfow** [reply]
NUBC/AHA for UB04 has information about Revenue Codes, along with CMS sites.  So when changes are made/added, I reference NUBC manual (AHA), for example.   I am not aware of an acronym, as either Rev Code or Revenue Code is usually used when referencing

**Reply by crsevern** [reply]
I want to echo the need here for rev codes. We worked with a lot of hospitals with rev code based rates during the provider disclosure.
"RC" is a standard abbreviation, though these also appear as "NUBC" on 837 files.

Example rate structures that would be "rev code only": 

- Inpatient per diem rate: RC 201 $5302/day
- High cost drug carveout: RC 636 paid at 36% of billed charges (separately, the payer may choose to disclose the specific drug at the JCode level, but disclosing % based rates for rev codes would be extremely useful for making sense of rate disclosures)

Edit - will also just add an extremely high volume of hospital contracts are Rev Code based.

**Reply by shaselton-usds** [reply]
Just a quick clarification here: when you say that hospital contracts are Rev Code based, is that in the context of an issuer/payer plans for consumers/patients?

Additional question: is the UB04 considered the version or considered to be the billing form (or maybe both) for a revenue code? e.g. was an earlier version called something like UB03?

I ask because this might be satisfied with the following:
```
"billing_code_type": "RC",
"billing_code_type_version": "UB-04",
"billing_code": "201"
```

**Reply by tdfow** [reply]
AHA publishes the UB-04 Data Specifications manual yearly, so the UB-04 has the billing information that is adopted by the NUBC, so I would think the version would need to include the year.

**Reply by vfohl** [reply]
UB-04 is the name of the claim form that facilities use. Previous to UB-04 it was UB-92 (UB-04 came out 2007ish),
For a contract example: Provider A has a per diem contract based on revenue code alone (no matter what CPT or HCPCS they may bill along with the rev code).
Rev code 124/126/128 pay  $X per diem
Rev code 1001/1002 pay $Z per diem
Any other rev code - not payable per contract and denies.

**Reply by nmayle** [reply]
It isn't uncommon for the charge to vary based on not only the HCPCS or CPT which identifies the procedure, but also depending on where the procedure is done in the hospital or other facility.  The revenue code defines where in the hospital the procedure happens.  In those cases, both a revenue code and a HCPCS would be needed.  So, the current design which allows only one billing code (corresponding to one billing code type) would be insufficient.  This also comes up with hospital administered drugs where there is both a HCPCS code and an NDC code.  You will need to modify the design to allow multiple codes for a single item.  Multiple codes (HCPCS + revenue_code) is a very common occurrence in the current price transparency files for hospitals.

**Comment by shaselton-usds** [comment]
Just wanted to follow up on this @tdfow -- did you happen to have an example that we can take a peek at? 

I'll probably close this in a few days if we can't think of anything.

**Comment by tdfow** [comment]
@shaelton-usds  I have reached out to see if I could get an example---but the scenarios I think fall into this category are related to where a Rev Code may be the basis for a given rate that is associated with where based on the Rev Code there may be a set rate---so how would this be reported?   Even in the mandate, it refers to see IG on how to report, thus the question.

**Reply by shaselton-usds** [reply]
You're definitely right that much of the details were deferred to the IG which is why I'm trying to get a bit more information about the specific scenario you have to help out :). 

As you know, the total set of reporting, codes, standards, etc are very different across industry and any attempt to have a solidified definition upfront would be a mistake as new information is learned.

When you get an example, please let me know and we can see if it'll fit the current schema's flexibility or if some alterations will need to be made.

**Reply by tdfow** [reply]
NUBC approves the Revenue Code values.   Here are some examples that could be associated with a negotiate rate based on Revenue Code, for example inpatient related billing:  Revenue Code 0361 (operating room service -minor surgery) , along with other Revenue Codes such as 0652 (Hospice, continuous home care), 0124 (room & board, semi-private- psychiatric), 0116 (room & board, private-detoxification), 126 (room & board, semi-private-detoxification), 128 (room & board, semi-private-rehab)

**Reply by rgekhter** [reply]
To add to this this: Unlike Medicare/Medicaid, Many commercial payers routinely contract for services based on revenue codes.  It’s not unusual to see per-visit reimbursement for outpatient Radiologoy/ER/Observation, or inpatient Per-Diem based on revenue code Room and Board codes.  And since there really isn’t a way to convert those into a Procedure Code, the spec should be extended to include Revenue Codes.

**Reply by rgouldkc** [reply]
We are a commercial rental PPO Network (re-pricer) and are working on this to feed to our payers that actually adjudicate and pay the claims. I'm looking at a specific contract for a hospital, For outpatient claims the rate can be determined by the service line procedure code only, the revenue code only and the combination of revenue code and service line procedure code. For inpatient rates can be determined by revenue code, revenue code and primary procedure code, and revenue code and primary diagnosis code. For this contract this part of the pricing process occurs after the claim is check for Stop-Loss, then DRG. If the service falls thru all of the above the discount is a percent off billed charges. So it's actually a little more complicated than just identifying a revenue code or a procedure code as some services will use a combination to derive a rate.

**Comment by tdfow** [comment]
regarding the billing code values, Multiple DRG types #47, also has a question on whether DRG should be replaced with MS-DRG and APR-DRG as billing code values or whether the type of DRG would be expected as part of the billing_code_type_version.

**Comment by tdfow** [comment]
How should o	APCs or "Ambulatory Payment Classifications" be reported?  Would there need to be a new billing code value for this?  This is used in bundled payment arrangements (outpatient)

**Reply by tdfow** [reply]
#61 is related to this question, seems to be posted on same day.

**Reply by tdfow** [reply]
Will there be a APC billing code value added to the schema or how should these be reported?

**Reply by tdfow** [reply]
@shaselton-usds Any CMS updates/comments regarding updating the schema to address either the Rev Code or APC billing code values?  If not, then how should these be published?

**Reply by iGokul-G** [reply]
@tdfow , @shaselton-usds  i totally agree,  Along with APC codes, we also use other  attributes like OPPS payment status code ( SI )  to drive pricing Addendum B OPPS 

Condition codes , discharge status codes could also control certain pricing which may also need representation

**Comment by shaselton-usds** [comment, marked as answer]
@tdfow 

The billing codes are not restricted to values found in the github documentation. An update can be found on this commit: https://github.com/CMSgov/price-transparency-guide/commit/55c96d0832baef07f8b0dbe1bd9cd945561ad0ab

The final rule helps out with this question with "...or other common payer identifier." - https://www.federalregister.gov/d/2019-25011/p-622

The exercise is to make sure that we standardize whatever common billing code is being used across the ecosystem. We've added a table at the bottom of the [documentation here](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#additional-notes-concerning-billing_code_type). As new codes are identified, they can be added there.

**Reply by taylorpatriciab** [reply]
@shaselton-usds The table very helpful - thank you.  Would you consider adding the following:  “EAPG” (Enhanced Ambulatory Patient Grouping) , ”LOCAL” (Some health plans assign local code for processing), “HIPPS” (Health Insurance Prospective Payment System), and “CDT” (Current Dental Terminology)

**Reply by taylorpatriciab** [reply]
@shaselton ICD is defined in the schema but I don't see it in the table

**Reply by shaselton-usds** [reply]
@taylorpatriciab I just want to double check that there are items/services that are negotiated based on these coding types.

**Reply by taylorpatriciab** [reply]
@shaselton-usds Yes, confirm that these are items / services that are negotiated based on these coding types.

**Reply by shaselton-usds** [reply]
@taylorpatriciab It has been mentioned here that negotiations don't happen at the ICD level - does this hold true for you?

It should be removed from the schema if this is the case.

**Reply by taylorpatriciab** [reply]
@shaselton-usds I'm not aware of negotiations at the ICD level.  I'm ok removing it.

**Reply by tdfow** [reply]
I had someone mentioned there was a scenario for diagnosis class or potential rate based on a diagnosis value, so there may be a need for ICD--for this scenario.

**Reply by ghost** [reply]
@tdfow A diagnosis can be used for the pricing of a claim that in turn can drive a negotiated rate. But that would be in the same sense like the charge, unit or a modifier. We do not believe, checked with our business SMEs, a ICD is a billing code type driver like CPT, HCPCS, DRG, APC and the others are can be found at the root level in agreements and terms for a provider for a negotiated rate.

**Reply by rgekhter** [reply]
An ICD-10 can and is used to contract with providers, but it's typically in conjunction with other services. So your main rate drive, for example could be a procedure, but ICD-10 is used as a comorbidity adjustment criteria.
ICD-10 PCX is a very different story.  Payers do on occasion write their contracts where ICD-10 PCX is the main driver for reimbursment. IE: in transplant cases, organ acquisition, complex procedures, etc...

**Reply by rgouldkc** [reply]
We also used ICD-10 to contract with providers. It is mostly for inpatient but can and is used for outpatient. Typically the rate has to be determined by a combination of Revenue Code, Procedure Code, and Diagnosis Code either singly or in some combination such as Revenue Code and Diagnosis Code.

**Reply by tdfow** [reply]
One of the scenarios provided that indicated they had pricing needs for ICD was related around eating disorders.  There was also mention where they had pricing based on some combinations, that included some ICD codes, so feels like it should be in the list if there are situations that would need to be supported.

**Reply by tdfow** [reply]
Regarding when there may be more than one billing code value that drives the price, do we assume these to be considered 'bundles', where then we would list the different billing codes that identify that rate?  If so, then would this be where if HCPCS and Rev Code were used to determine a rate, we report as 'bundle' and provide HCPCS and RC codes that make up that rate?  If there were a CPT, ICD type combination, then would the ICD value need to be a valid billing code to use in a bundle scenario?

**Reply by taylorpatriciab** [reply]
Does a distinction need to be made between a negotiated rate and the application of benefits to determine cost sharing?  For example, if the diagnosis is cancer, the patient cost sharing may be waived.  In this example, would that be an application of benefits, but it does not affect the negotiated rate?

**Reply by taylorpatriciab** [reply]
@shaselton-usds it sounds like there are pricing arrangements based on ICD codes.  May it be added back?

**Reply by shaselton-usds** [reply]
@taylorpatriciab Added in. https://github.com/CMSgov/price-transparency-guide/commit/3cc35a30cb6032f891d5f3dde4da7305ee88766a

**Reply by taylorpatriciab** [reply]
@shaselton-usds .  thank you.  would you add: “EAPG” (Enhanced Ambulatory Patient Grouping) , ”LOCAL” (Some health plans assign local code for processing), “HIPPS” (Health Insurance Prospective Payment System), and “CDT” (Current Dental Terminology)

**Reply by shaselton-usds** [reply]
@taylorpatriciab Would it be possible to open a pull request with those values?

**Reply by tdfow** [reply]
@taylorpatriciab @shaselton-usds I did open a pull request for the in-network/allowed amount schema references

**Reply by shaselton-usds** [reply]
@taylorpatriciab Do you have cases where your negotiated rates are determined via LOCAL? Possible to provide some examples of what LOCAL look like with a `billing_code_type`, `billing_code_type_version`, and `billing_code` example?

@tdfow Thanks for the PR -- I'll make some comments there.

**Reply by taylorpatriciab** [reply]
@shaselton-usds There are a couple of examples:
1)  The payer assigns procedure codes that aren’t part of the CPT / HCPCS code system.  "billing_code_type":"LOCAL","billing_code_type_version":"2021","billing_code":"12345"
2)  The payer assigns a code based on an APC-like grouper that does not tie to standard bill code.  The reimbursement is a factor of a category of services rendered using revenue code, patient condition and diagnosis codes to determine provider reimbursement.  
"billing_code_type":"LOCAL","billing_code_type_version":"2021","billing_code":"abcde"

**Reply by nicholas-dahl-CTS** [reply]
@shaselton    If a system doesn't specify sub-types of DRGs, but just labels them generically as "DRGs" is that supported?  There is not an entry for that as a code type in the table
